frontend/app/routes/protected/profile/applicant-information.tsx

```diff
@@ -80,16 +99,16 @@ export default function ProtectedApplicantInformation({ loaderData, params }: Ro
       </dl>
       {children.map((child) => {
         return (
-          <dl className="divide-y border-y" key={child.information.id}>
-            <h2 className="font-lato py-4 text-2xl font-bold">{`${child.information.firstName} ${child.information.lastName}`}</h2>
+          <dl className="divide-y border-y" key={child.id}>
+            <h2 className="font-lato py-4 text-2xl font-bold">{`${child.firstName} ${child.lastName}`}</h2>
```

Comment:
Move the heading outside of the definition list and wrap the heading and definition list with section

```suggestion
          <section className="space-y-6">
            <h2 className="font-lato py-4 text-2xl font-bold">{`${child.firstName} ${child.lastName}`}</h2>
            <dl className="divide-y border-y" key={child.id}>  
```